tests/test_meshtastic_utils.py

```
@@ -4772,22 +4742,25 @@ def test_disconnect_ble_by_address_disconnect_timeout_logs_warning(
         from mmrelay.meshtastic_utils import _disconnect_ble_by_address
 
-        async def _noop(*_args, **_kwargs):
-            """
-            Asynchronous no-op that ignores all positional and keyword arguments.
-
-            Returns:
-                None: Always returns None.
-            """
-            return None
+        class _ImmediateAwaitable:
+            def __await__(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__await__`

(ANN204)